<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinaryWatchWidget</class>
 <widget class="QWidget" name="BinaryWatchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>238</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Binary Watch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Hours</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="hour8Led">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hour4Led">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hour2Led">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hour1Led">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>AM/PM</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>AM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="amLed">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>PM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pmLed">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Minutes</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="minute32Led">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minute16Led">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minute8Led">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minute4Led">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minute2Led">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minute1Led">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Seconds</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="second32Led">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="second16Led">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="second8Led">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="second4Led">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="second2Led">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="second1Led">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="led_pic.qrc">:/icons/ledgrey.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="led_pic.qrc"/>
 </resources>
 <connections/>
</ui>
